test/integration_test.py

```diff
@@ -1244,9 +1245,9 @@ def file_name(part):
 
     def test_can_bundle_configured_catalogs(self):
         for catalog_name, catalog in config.catalogs.items():
-            if catalog.is_integration_test_catalog:
```

Comment:
... and this, all in a separate commit so the history looks like this:

1) Add local plugin
2) Add cans
3) Move cans and local plugin to attic
4) Main commit